### ISSUE TITLE
GPS jumped around

### ISSUE THREAD

**Issue by PatriciaMoller** [SmartZoos/Toolset#114]
During the testing at Skansen 16/5, we experienced that the GPS position was jumping back-and-forth:
![gps error](https://cloud.githubusercontent.com/assets/26867575/26240046/e40f2ae2-3c7f-11e7-9870-2738d507dc63.png)

Is this purely due to connection issues to Google Maps, or can it be caused by something on SmartZoos'/our end?

Discovered on:
iPhone 7 Plus
Samsung Galaxy J5, 6.0.1

**Comment by kaimikael** [comment]
I see that happening quite frequently with other geo-apps, so it's not our fault.  
One work-around would be to filter positions, so that readings that bounce away like that are discarded.

**Comment by pjotrsavitski** [comment]
Yes, things like that can happen. Geopositioning accuracy depends on multiple factors. We could determine the standard location, but the logic to accept or disregard the change is not so simple. Example: no new positioning information was received for 3 minutes and the difference could be a few hundred meters. This one is just from the top of my head, there are surely many more factors. We just need to consider sane possibilities.

**Comment by kaimikael** [comment]
Usually Kalman filters are recommended for smoothing out noise like this, but I'm not sure what the behaviour will be with a time constant of several minutes.  
This issue is probably not a high priority, but if we get time in the future we could look into what filtering can give.
